packages/auth-client/src/index.ts

```diff
@@ -342,26 +348,7 @@ export class AuthClient {
    *  }
    * });
    */
-  public async login(options?: {
-    /**
-     * Identity provider
-     * @default "https://identity.ic0.app"
-     */
-    identityProvider?: string | URL;
-    /**
-     * Expiration of the authentication in nanoseconds
-     * @default  BigInt(8) hours * BigInt(3_600_000_000_000) nanoseconds
-     */
-    maxTimeToLive?: bigint;
-    /**
-     * Callback once login has completed
-     */
-    onSuccess?: (() => void) | (() => Promise<void>);
-    /**
-     * Callback in case authentication fails
-     */
-    onError?: ((error?: string) => void) | ((error?: string) => Promise<void>);
-  }): Promise<void> {
+  public async login(options?: AuthClientLoginOptions): Promise<void> {
```

Comment:
Not sure why this was duplicated.

Reply:
Duplicating the type allows for a more expansive tooltip in editors. Try hovering over an initialized `authClient.login` in VSCode with both versions and you'll get a sense of it.

Reply:
Oh, yea good point. Hopefully we get this expansion automatically soon. Reverted the change. Thanks for the explanation 🙏